docs/DeveloperGuide.md

```diff
@@ -114,21 +114,24 @@ How the parsing works:
 * All `XYZCommandParser` classes (e.g., `AddCommandParser`, `DeleteCommandParser`, ...) inherit from the `Parser` interface so that they can be treated similarly where possible e.g, during testing.
 
 ### Model component
-**API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
+**API** : [`Model.java`](https://github.com/AY2223S1-CS2103-F14-3/tp/blob/bb7ac45ed934914d1814ef862edaddf4ccfff320/src/main/java/seedu/application/model/Model.java)
```

Comment:
Probably shouldn't use the permalink here. This link always directs to a particular commit in the past (the one with hash bb7ac45...) The link that always goes to the latest version can be taken from the address bar

```suggestion
**API** : [`Model.java`](https://github.com/AY2223S1-CS2103-F14-3/tp/blob/master/src/main/java/seedu/application/model/Model.java)
```